README.md

```diff
@@ -1,5 +1,20 @@
 
-# Contributing
+# Azure Pipelines Kubernetes UI
+
+## Overview
+
+This repo contains React based UI view of a kubernetes cluster and will be used in azure devops pipelines. This UI is hostable outside of azure pipelines product and does not require the UI web server to be running inside the kubernetes cluster.
+
+This repo consists of
+1. Contracts: The `IKubeService` provides the interface that needs to be implemented inorder to fetch data needed for the UI
```

Comment:
inorder --> in order

if possible add dependecies like vssui, sdk, api and k8s client